contracts/SimpleMarginTrading.sol

```diff
@@ -116,66 +116,69 @@ contract SimpleMarginTrading
+        positionBalance.safeAdd(fillResults.makerAssetFilledAmount);
+        borrowBalance = Cdai.borrowBalanceCurrent(address(this));
+        // at this point you have ceth, and swapped for WETH
     }
 
     function close(
        ZeroExQuote memory quote
     )
         public
-        onlyOwner
+        onlyowner
         onlyWhenOpen
         returns (uint ethBalance)
     {
         // 1. approve for swap
-        _approve(address(WETH), _getZeroExApprovalAddress());
+        _approve(address(weth), _getZeroExApprovalAddress());
         // 2. verify swap
-        uint256 wethBalance = WETH.balanceOf(address(this));
-        uint256 daiBorrowBalance = CDAI.borrowBalanceCurrent(address(this)); // TODO
+        uint256 wethBalance = weth.balanceOf(address(this));
+        uint256 daiBorrowBalance = Cdai.borrowBalanceCurrent(address(this)); // TODO
         require(wethBalance > quote.sellAmount, "not enough to swap");
-        require(quote.buyToken == address(DAI), "not buying DAI");
-        require(daiBorrowBalance < quote.buyAmount, "not enough DAI to repay");
+        require(quote.buyToken == address(dai), "not buying dai");
+        require(daiBorrowBalance < quote.buyAmount, "not enough dai to repay");
         // 3. execute swap
-        (bool success, bytes memory data) = address(EXCHANGE).call.value(quote.protocolFee)(quote.calldataHex);
+        (bool success, bytes memory data) = address(exchange).call.value(quote.protocolFee)(quote.calldataHex);
         require(success, "Swap not filled.");
         // 4. decode results
         LibFillResults.FillResults memory fillResults = abi.decode(data, (LibFillResults.FillResults));
-        // 5. return back DAI
-        _approve(address(DAI), address(CDAI));
-        require(CDAI.repayBorrow(fillResults.makerAssetFilledAmount) == 0);
+        // 5. return back dai
+        _approve(address(dai), address(Cdai));
+        require(Cdai.repayBorrow(fillResults.makerAssetFilledAmount) == 0, "repayment of dai to Compound failed");
         // 6. get back ETH
-        require(CETH.redeem(CETH.balanceOfUnderlying(address(this))) == 0); // TODO correct?
-        // 7. withdraw all WETH Balance;
-        WETH.withdraw(wethBalance);
+        require(ceth.redeem(ceth.balanceOfUnderlying(address(this))) == 0, "withdraw of ETH from Compound failed"); // TODO correct?
+        // 7. withdraw all weth Balance;
+        weth.withdraw(wethBalance);
         // 8. transfer all ETH back to owner;
         ethBalance = address(this).balance;
-        OWNER.transfer(address(this).balance);
+        owner.transfer(address(this).balance);
         // 9. reset balance
         positionBalance = 0;
     }
 
-    // handy function to get borrowed DAI amount
-    function getBorrowBalance() public onlyOwner returns (uint256) {
-        return CDAI.borrowBalanceCurrent(address(this));
+    // handy function to get borrowed dai amount
+    function getBorrowBalance() public onlyowner returns (uint256) {
```

Comment:
Use the same style with all of the functions in this file. This is all on one line, but the others are spread across multiple lines. Use the other style for this function.